frontend/src/components/volunteer/VolunteerNavbar.tsx

```diff
@@ -21,6 +24,13 @@ const VolunteerNavbar = ({
 }: {
   defaultIndex: VolunteerPages;
 }): React.ReactElement => {
+  const {
+    authenticatedUser,
+  }: { authenticatedUser: AuthenticatedUser } = useContext(AuthContext);
+  const userName = !authenticatedUser
+    ? "Sistering Volunteer"
+    : `${authenticatedUser.firstName} ${authenticatedUser.lastName}`;
```

Comment:
```suggestion
  const userName =`${authenticatedUser?.firstName} ${authenticatedUser?.lastName}`;
```
In practice, users who are not authenticated shouldn't be able to access this page. So we can just use the optional chain operator. 

You don't need to store it in variable since it's simplified now.